docs/utilities.rst

```diff
@@ -3,6 +3,30 @@ Utilities
 
 TODO: update
 
+GUI system
+----------
+
+.. code-block:: python
+
+    gui = ti.GUI('Title', (640, 480))
+    while not gui.is_pressed(ti.GUI.ESCAPE):
+        gui.set_image(img)
+        gui.show()
+
+
+Also checkout https://github.com/taichi-dev/taichi/blob/master/examples/keyboard.py for more advanced event processing.
```

Comment:
TODO: we want a more detailed API desc like atomic and vector does.

Reply:
We can always do a simple tutorial plus a complete API reference. For `GUI` this also applies :-)